app/views/playlist.js

```diff
@@ -3,8 +3,18 @@ import application from '../application'
 
 const PlaylistView = Mn.ItemView.extend({
 
+  template: require('./templates/popupPlaylists'),
```

Comment:
My mistake, the actual template I wanted to require is the `./templates/playlist`.